### ISSUE TITLE
Save log path not set on iOS

### ISSUE THREAD

**Issue by dogmaphobic** [mavlink/qgroundcontrol#7173]
![img_0008](https://user-images.githubusercontent.com/749243/51443881-c3bf4680-1cbd-11e9-8c42-ecc7e8c27cf5.PNG)

**Comment by DonLakeFlyer** [comment]
Built master and it seemed to work fine for me?

**Comment by dogmaphobic** [comment]
I created the issue so I would not forget. I can't test it now. Did you have QGC already installed when you "updated" with a new master build?

**Comment by DonLakeFlyer** [comment]
I started with no build.

**Comment by DonLakeFlyer** [comment]
I didn't specifically try telemetry log, but Mission save worked fine.

**Comment by dogmaphobic** [comment]
Yeah, I would need to try again and see what was going on. That will have to wait...

**Comment by DonLakeFlyer** [comment]
Maybe a problem with settings reset? Did you get the settings blown away prompt?
